### PR TITLE
Fixed executing DA-CLI from arbitrary folder

### PR DESCRIPTION
**Pull request by stanlyDoge**

### CODE REVIEW

**Comment by janinko** [comment]
2 changes in one commit?

**Comment by stanlyDoge** [comment]
renaming is related to import which has been modified so did not consider it as appropriate to split to two commits

File: cli-wrap/src/main/resources/cli/da-cli.py [hunk 1]
```diff
@@ -1,7 +1,13 @@
...
 import sys
-from listings import *
 import argparse
+import inspect
+import os
+
+cmd_folder = os.path.realpath(os.path.abspath(os.path.split(inspect.getfile( inspect.currentframe() ))[0]))
+if cmd_folder not in sys.path:
+     sys.path.insert(0, cmd_folder)
+
```
Comment (janinko) [inline cli-wrap/src/main/resources/cli/da-cli.py]: Why are you inserting the path to sys.path? can't it be done just by prepending the path to config file name?

**Comment by janinko** [comment]
and why did you renamed the import?

File: cli-wrap/src/main/resources/cli/da-cli.py (hunk 1 above)
Comment (stanlyDoge) [inline cli-wrap/src/main/resources/cli/da-cli.py]: because other script files has some custom files import too and code to reach import them is ugly

**Comment by stanlyDoge** [comment]
I renamed import because it was named pm (python module) which is not very good describing. Should I revert this change?

**Pull request merged**